models/intel/smartlab-object-detection-0004/model.yml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  Mobilenet-v2 transfer-learned on mythware dataset. Origianl classifier layer is dropped
+task_type: action_recognition
+files:
+  - name: FP32/smartlab-object-detection-0001.xml
```

Comment:
you should not provide model.yml files for models, which should be provided as intel trained, these files are automatically generated when models uploaded for release

Reply:
removed